ONSdigital/dp-frontend-renderer
# added logic to ensure no trailing comma in json-ld download section

### What

Implemented logic to remove trailing commas in the JSON-LD downloads section. This will ensure the JSON-LD is valid

### How to review

- Check code makes sense
- Run the renderer with `ENABLE_JSONLD_CONTROL` set to `true` and ensure the frontend dataset controller is running too. Navigate to a legacy or filterable dataset landing page that contains download links and check that the downloads section of the JSON LD tag is valid

### Who can review

Anyone but me
